book/book.toml

```diff
@@ -0,0 +1,28 @@
+[book]
+authors = ["Josh Rotenberg"]
+language = "en"
+multilingual = false
+src = "src"
+title = "Clippy Documentation"
+
+[rust]
+edition = "2018"
+
+[output.html]
+edit-url-template = "https://github.com/rust-lang/rust-clippy/edit/master/guide/{path}"
+git-repository-url = "https://github.com/rust-lang/rust-clippy/tree/master/guide"
```

Comment:
I think those URLs are outdated?

Reply:
See below. I fixed them, but we can also remove them.